openedx/core/djangoapps/content_libraries/api/containers.py

```diff
@@ -372,7 +373,7 @@ def restore_container(container_key: LibraryContainerLocator) -> None:
 def get_container_children(
     container_key: LibraryContainerLocator,
     published=False,
```

Comment:
This is unrelated to this PR, but a bad conflict merge broke [this line](https://github.com/openedx/edx-platform/pull/36553/files#diff-5a248b18ae17c86d454776cb48e0372a8cbdc9610b3886f175a116f9b280e681R346) above. Could you fix this here?

```diff
@@ -343,7 +343,7 @@ def restore_container(container_key: LibraryContainerLocator) -> None:

     LIBRARY_CONTAINER_CREATED.send_event(
         library_container=LibraryContainerData(
-            container_key=str(container_key),
+            container_key=container_key,
         )
     )
```

Reply:
Sure.